nl_server/spacy_model.py

```diff
@@ -11,25 +11,25 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""Managing the NER model for place detection."""
+"""Managing the Spacy model for NL detection."""
 
 from typing import List
 
 import en_core_web_lg
 
 
-class NERPlaces:
+class SpacyModel:
```

Comment:
From reading the documentation, seems NER is one function of Spacy. In this change, since I am detecting the sentence structure, it's not for NER anymore, that's the reason for renaming.

I agree using SpacyModel is too specific to implementation, how about using NLModel (not sure if that's too general..)

Reply:
How about QueryAttributeModel? NLAttributeModel? QueryNLAttributeModel? Something along those lines?